db - fails to test

```
$ git log --oneline HEAD^.. --no-decorate 
078626754 pixi/0.43.3 package update (#48168)

$ make test/db                                                              
Testing package db with version db-5.3.28-r2 from file db.yaml                                               
/disks/home/smoser/go/bin/melange test db.yaml --repository-append /disks/home/smoser/src/wolfi-os/packages -
-keyring-append local-melange.rsa.pub --arch x86_64 --pipeline-dirs ./pipelines/ --repository-append https://
packages.wolfi.dev/os --keyring-append https://packages.wolfi.dev/os/wolfi-signing.rsa.pub --test-package-app
end wolfi-base --debug  --source-dir ./db/                                                                   
2025/03/25 16:06:50 INFO building test workspace in: '/tmp/melange-guest-3221497981-main' with apko
2025/03/25 16:06:50 INFO image configuration:                                                                
2025/03/25 16:06:50 INFO   contents:                                                                         
2025/03/25 16:06:50 INFO     build repositories: []                                                          
2025/03/25 16:06:50 INFO     runtime repositories: []                                                        
2025/03/25 16:06:50 INFO     keyring:      []                                                                
2025/03/25 16:06:50 INFO     packages:     [db apk-tools busybox posix-libc-utils]          
2025/03/25 16:06:50 INFO   accounts:                                                                         
2025/03/25 16:06:50 INFO     runas:                                                                          
2025/03/25 16:06:50 INFO     users:                                                                          
2025/03/25 16:06:50 INFO       - uid=1000(build) gid=1000                                   
2025/03/25 16:06:50 INFO     groups:                                                                         
2025/03/25 16:06:50 INFO       - gid=1000(build) members=[build]
...
2025/03/25 16:06:53 INFO running step "docs readability check"
2025/03/25 16:06:53 WARN + '[' -d /home/build ]
2025/03/25 16:06:53 WARN + cd /home/build
2025/03/25 16:06:53 WARN + basename /home/build/melange-out/db-doc
2025/03/25 16:06:53 WARN + doc_pkg=db-doc
2025/03/25 16:06:53 WARN + apk info -qL db-doc
2025/03/25 16:06:53 WARN + grep -v ^var/lib/db/sbom
2025/03/25 16:06:53 WARN + grep -v '^$'
2025/03/25 16:06:53 WARN + wc -l
2025/03/25 16:06:53 WARN + '[' 5245 -eq 0 ]
2025/03/25 16:06:53 WARN + cd /
2025/03/25 16:06:53 WARN + doc_files=false
2025/03/25 16:06:53 WARN + apk info -qL db-doc
2025/03/25 16:06:53 WARN + grep ^usr/share/man/
2025/03/25 16:06:53 WARN + '[' -f /usr/share/man/db/index.html ]
2025/03/25 16:06:53 WARN + man -l /usr/share/man/db/index.html
2025/03/25 16:06:53 WARN troff:<standard input>:49: warning [p 1, 6.5i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:65: warning [p 1, 8.5i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:87: warning [p 2, 0.0i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:95: warning [p 2, 1.2i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:110: warning [p 2, 2.8i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:111: warning [p 2, 3.0i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:133: warning [p 2, 5.5i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:141: warning [p 2, 6.7i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:169: warning [p 2, 10.2i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:190: warning [p 3, 1.3i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:193: warning [p 3, 2.0i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:214: warning [p 3, 4.3i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:222: warning [p 3, 5.5i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:230: warning [p 3, 6.5i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:230: warning [p 3, 6.7i]: cannot break line
2025/03/25 16:06:53 WARN troff:<standard input>:238: warning [p 3, 7.7i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:246: warning [p 3, 8.7i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:246: warning [p 3, 8.8i]: cannot break line
2025/03/25 16:06:53 WARN troff:<standard input>:254: warning [p 3, 9.8i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:269: warning [p 4, 0.3i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:270: warning [p 4, 0.5i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:287: warning [p 4, 2.8i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:299: warning [p 4, 4.3i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:300: warning [p 4, 4.5i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:317: warning [p 4, 6.3i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:318: warning [p 4, 6.8i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:325: warning [p 4, 7.7i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:326: warning [p 4, 8.0i]: cannot adjust line
2025/03/25 16:06:53 WARN troff:<standard input>:328: warning [p 4, 8.3i]: cannot adjust line
2025/03/25 16:06:53 WARN + doc_files=true
2025/03/25 16:06:53 WARN + '[' -f /usr/share/man/db/api_reference/C/BDB-C_APIReference.pdf ]
2025/03/25 16:06:53 WARN + man -l /usr/share/man/db/api_reference/C/BDB-C_APIReference.pdf
2025/03/25 16:06:53 WARN man: iconv_open ("UTF-8//IGNORE", "ISO-8859-1"): Invalid argument
2025/03/25 16:06:53 WARN grotty:<standard input>:(<standard input>):7: fatal error: 'V' command invalid before first 'p' command
2025/03/25 16:06:53 WARN man: command exited with status 1: /usr/libexec/man-db/zsoelim | /usr/libexec/man-db/manconv -f UTF-8:ISO-8859-1 -t UTF-8//IGNORE | preconv -e UTF-8 | tbl | nroff -mandoc -rLL=79n -rLT=79n -Tascii
2025/03/25 16:06:53 ERRO ERROR: failed to test package. the test environment has been preserved:
2025/03/25 16:06:53 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: exit status 3
make: *** [Makefile:120: test/db] Error 1

$ melange version
  __  __   _____   _          _      _   _    ____   _____
 |  \/  | | ____| | |        / \    | \ | |  / ___| | ____|
 | |\/| | |  _|   | |       / _ \   |  \| | | |  _  |  _|
 | |  | | | |___  | |___   / ___ \  | |\  | | |_| | | |___
 |_|  |_| |_____| |_____| /_/   \_\ |_| \_|  \____| |_____|
melange

GitVersion:    v0.23.1
GitCommit:     unknown
GitTreeState:  unknown
BuildDate:     unknown
GoVersion:     go1.23.7
Compiler:      gc
Platform:      linux/amd64
```

--- a/pipelines/test/docs.yaml
+++ b/pipelines/test/docs.yaml
@@ -32,7 +32,9 @@ pipeline:
       cd /
       doc_files=false
       # Test man pages
-      for doc_file in $(apk info -qL "$doc_pkg" | grep "^${{inputs.path-prefix}}/man/"); do
+      # Exclude things installed under /usr/share/man/db/ since it contains
+      # files installed by the man-db-doc package.
+      for doc_file in $(apk info -qL "$doc_pkg" | grep "^${{inputs.path-prefix}}/man/" | grep -v "^usr/share/man/db/"); do
         if [ -f /"$doc_file" ]; then
           # Ensure that man can read and render
           # NOTE: man will dutifully, print any text file, not just troff manpages (e.g. html or plain text too)

